Dockerfile

```diff
@@ -14,7 +15,15 @@ RUN yum update -y && yum install -y wget \
   && cp -rf apache-maven-3.2.5 /opt/maven && ln -fs /opt/maven/bin/mvn /usr/bin/mvn \
   && mvn package -Dmaven.test.skip=true \
   && mv target/perseo-core-*.war /usr/share/tomcat/webapps/perseo-core.war \
-  && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo
+  && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo \
+  && echo "# This file should be copied by deployment process" > /etc/perseo-core.properties \
+  && echo "# into /etc/perseo-core.properties, with the appropiate permissions" >>  /etc/perseo-core.properties \
+  && echo "" >>  /etc/perseo-core.properties \
+  && echo "# URL for invoking actions when a rule is fired" >>  /etc/perseo-core.properties \
+  && echo "action.url = http://${PERSEO_FE_URL}/actions/do" >>  /etc/perseo-core.properties \
+  && echo "" >>  /etc/perseo-core.properties \
+  && echo "# Time in milliseconds (long) to "expire" a "dangling" rule" >>  /etc/perseo-core.properties \
```

Comment:
I understand that quotes (") surrounding 'expire' and 'dangling' should be escaped.